api/utils.py

```diff
@@ -8,3 +8,20 @@ def get_language_from_filename(filename):
         return "chn"
     else:
         return "pli"
+
+
+def create_teststring(limitcollection,language):
```

Comment:
get we please get a better description of this method? Having a docstring would be really nice.